### DISCUSSION TITLE
missing modified phosphorylated threonine residue (THP1)

### DISCUSSION THREAD

**Discussion by sau1978** [Electrostatics/pdb2pqr#328] [Q&A]
Hello 
 I am using the web interface to produce the PQR AND APBS files, to visualization in VMD.  My structure has several phosphorylated serines and phosphorylated threonines. After adding the parameters for both (SP2 and THP1) to the CHARMM.DAT file I am able to run on the web interface.  Afterwards, I am able to visualize the phosphorylated serines, but the phosphorylated threonines (residues 382 and 383) are missing (designated THP1).
What can I do to create a PQR file that also includes the missing phosphorylated threonine residues?
<img width="962" alt="Screen Shot 2022-09-02 at 11 01 35 AM" src="https://user-images.githubusercontent.com/26680556/188178057-f803d293-bfe0-4125-893d-003af35d7c97.png">

**Comment by sobolevnrm** [comment]
Hello --

We have [a section](https://pdb2pqr.readthedocs.io/en/latest/extending.html) explaining how to do this in the [PDB2PQR documentation](https://pdb2pqr.readthedocs.io/en/latest/index.html).

Thank you

**Reply by sau1978** [reply]
Thank you for your quick response I followed the protocol in the link provided. However, after several attempts I am not sure why the SP2 residue works but not the THP1 residue. Is there a specific non-canonical CHARMM naming scheme for threonine which includes phosphorylated THP1? I have searched through discussion section and the web to accurately account for this residue, but have not had success. Thank you.

**Reply by sobolevnrm** [reply]
I am not sure if there's a specific CHARMM naming scheme for phosphothreonine.  Did you add the parameters to the CHARMM parameter file per the instructions in the documentation I mentioned above?  Are you able to share your modified files and any error messages you received?

**Comment by sau1978** [comment]
Thank you for your response. Yes, please see the attached CHARMM.DAT file which indicate the parameters for both the phosphoserine (SP2) and phosphothreonine (THP1). I am also including the pdb2pqr.stderror.txt file. I appreciate you taking the time to help me troubleshoot.
[pdb2pqr-phosphthr-error-jobid-wj4hhyaikp.zip](https://github.com/Electrostatics/pdb2pqr/files/9527872/pdb2pqr-phosphthr-error-jobid-wj4hhyaikp.zip)

**Reply by sau1978** [reply]
Good afternoon! I'm not sure if you have had the time to look at my error regarding the phosphothreonine (THP1) for jobid wj4hhyaikp. Thanks in advance for your attention to troubleshooting the issue.

**Comment by sobolevnrm** [comment]
I'm sorry but we're understaffed and I haven't had a chance to look at this yet.  If anyone else in the APBS/PDB2PQR community is able to help, it would be much appreciated.
